.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
I think we should pull this comment since this is json?

Reply:
Sure, I'll pull it.  For what it's worth, this is generated by VS Code and I believe all of the configs are parsed as `jsonc` (i.e. JSON with comments).